### DISCUSSION TITLE
BUG con el logro Galaxy Brain

### DISCUSSION THREAD

**Discussion by Jkeviin** [community/community#31932] [Other Feature Feedback, Questions, & Ideas] [answered]
Ya  he hecho respondido discusiones de otras personas, las cuales me dieron su "Respondido", muchas veces y no me dan el logro, no sé que pasa.

**Comment by DenverCoder1** [comment, marked as answer]
The badge is only earned by having *another user* mark your answer as accepted. If you select your own answer, it does not count for the achievement.

You can see discussions you answered here: https://github.com/search?q=answered-by%3AJkeviin

None of these seem like discussions that should count towards earning the achievement.

**Reply by Jkeviin** [reply]
https://github.com/Jkeviin2/Prueba-123/discussions/2

Ya hice eso, aun asi no me cumple, desde otro usuario creé la discusion, después con esta cuenta respondí esa discusión, por ultimo con el otro usuario acepté la respuesta. (3 veces)

Aún asi, no tengo el logro, lo intenté con dos cuentas nuevas y con esa si tengo el logro.

**Reply by DenverCoder1** [reply]
It is possible there is a filter on spam or new accounts to avoid users making fake answers to get the achievement.

You can try submitting a ticket to [GitHub Bug Report](https://support.github.com/contact/bug-report) over here and give them a few days to respond.

Hope this helps.
